### ISSUE TITLE
Is the code running without GPU?

### ISSUE THREAD

**Issue by liujiaozhuwudi** [divamgupta/deep_clustering_kingdra#9]
Is the code running without GPU?

**Comment by divamgupta** [comment]
If tensorflow is configured properly to use GPU then it will definitely use
GPU

**Comment by liujiaozhuwudi** [comment]
Thank you very much for your reply, I'll go find out why the GPU usage is 0.I saw in the question you replied that you mentioned that the version of sklearn will affect the accuracy. I would like to ask whether it has a great impact？

**Comment by liujiaozhuwudi** [comment]
Ok, I am close to reproducing the result but there is a problem of insufficient CPU memory. Do you have a pytorch implementation version of this ladder network?

**Comment by liujiaozhuwudi** [comment]
Thank you so much for your work! I did not achieve the ACC accuracy written in the paper on CIFAR-10 according to the hyperparameter settings you gave.
